source/base/parameter_handler.cc

```diff
@@ -1409,7 +1418,40 @@ bool ParameterHandler::read_input_from_string (const char *s)
 {
   // create an istringstream representation and pass it off
   // to the other functions doing this work
-  std::istringstream in (s);
+
+  // concatenate the lines ending with "\"
+
+  std::istringstream file(s);
+  std::string str;
+  std::string file_contents;
+  bool is_concatenated(false);
+  while (std::getline(file, str))
+    {
+      // if this str must be concatenated with the previous one
+      // we strip the whitespaces
+      if (is_concatenated)
+        {
+          const std::size_t str_begin = str.find_first_not_of(" \t");
+          if (str_begin == std::string::npos)
+            str = ""; // no content
+          else
+            str.erase(0,str_begin); // trim whitespaces from left
```

Comment:
Thanks for pointing it out. The file `parameter_handler_backslash_02.output` was the old version. In the new one there are no spaces. Yet the test was successful